draft-ietf-tls-tls13.md

```diff
@@ -3541,20 +3551,20 @@ C, and D.
 
 #  IANA Considerations
 
-[[TODO: Update https://github.com/tlswg/tls13-spec/issues/62]]
 [[TODO: Rename "RSA" in TLS SignatureAlgorithm Registry
 to RSASSA-PKCS1-v1_5 ]]
 
 This document uses several registries that were originally created in
 {{RFC4346}}. IANA has updated these to reference this document. The registries
-and their allocation policies (unchanged from {{RFC4346}}) are listed below.
+and their allocation policies are below:
 
--  TLS Cipher Suite Registry: Future values with the first byte in
-  the range 0-191 (decimal) inclusive are assigned via Standards
-  Action {{RFC2434}}.  Values with the first byte in the range 192-254
-  (decimal) are assigned via Specification Required {{RFC2434}}.
+- TLS Cipher Suite Registry: Values with the first byte in the range
+  0-254 (decimal) are assigned via Specification Required {{RFC2434}}.
   Values with the first byte 255 (decimal) are reserved for Private
-  Use {{RFC2434}}.
+  Use {{RFC2434}}. IANA [SHALL add/has added] a "Recommended" column
+  to the cipher suite registry. All cipher suites listed in
+  {{cipher-suites}} are marked as "Yes". All other cipher suites are
+  marked as "No".
 
```

Comment:
I'd really like the registry to include the rationale for the "yes"/"no".  It would appear above the available formats:

    IANA [SHALL add/has added] the following to the following to the TLS Cipher Suite Registry:
    
    NOTE
          Cipher suites marked as "Yes" are the MTI TLS 1.3 cipher suites
          in the RFC 2119-sense (See Section 8.1 of [THISRFC]) and can be
          updated later. Algorithms marked as "No" are not; cipher suites
          marked "No" range from "good" to "bad" from a cryptographic
          standpoint.